### ISSUE TITLE
Contracts: contracts date range

### ISSUE THREAD

**Issue by heldergg** [jorgecarleitao/public-contracts#75]
When we're exploring a given contract category it would be nice to be able to view the summary tab for a given date range. For instance, the "[Serviços de TI: consultoria, desenvolvimento de software, Internet e apoio (72000000-5)](http://publicos.pt/categoria/8215)" category has the following summary:

* 907.2 million € (18198 contracts)
* 1615 contractors
* 4001 contracted

I gather that the above numbers account all the contracts in the db of this category.

What I'm suggesting is to have a date range selector for the summary, or even a date range selector for all these tabs:  summary, contracts, contractors, contracted and tenders.

**Comment by jorgecarleitao** [comment]
I agree, and I also think that this could be generalised to have aggregations with filters over different fields (e.g. "date range + only `ajusto direto` + category X"). Maybe there is already a solution for this in Django's ecosystem.
